dev/io.openliberty.microprofile.faulttolerance.4.1.internal_fat_tck/fat/src/io/openliberty/microprofile/faulttolerance41/tck/FaultToleranceTck41Launcher.java

```diff
@@ -51,7 +51,7 @@ public class FaultToleranceTck41Launcher {
     private static final boolean FAT_TEST_LOCALRUN = Boolean.getBoolean("fat.test.localrun");
 
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP70);
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP70_EE10);
```

Comment:
Will we want to also do MP 70_EE11?

Reply:
yes but adding more repeats was going to be a separate PR ... or PRs per bucket